src/coreclr/tools/aot/ILCompiler.Diagnostics/PdbWriter.cs

```diff
@@ -90,6 +90,7 @@ public partial class PdbWriter
         Dictionary<SymDocument,int> _documentToChecksumOffsetMapping;
 
         UIntPtr _pdbMod;
+        IntPtr _pdbWriterInst;
```

Comment:
I think you can release the reference right after the GetOrCreateObjectForComInstance call. No need to store it in an extra instance field.

Reply:
Good point, thanks Jan for pointing that out, fixed in 2nd commit.